tangelo/linq/tests/test_ibm_connection.py

```diff
@@ -0,0 +1,80 @@
+"""
+    A test class to check that features related to IBM API for quantum experiments are behaving as expected.
+    Tests requiring actual interactions with the services are not run automatically.
+"""
+
+import unittest
+import os
+import time
+
+from tangelo.linq import Gate, Circuit
+from tangelo.linq.qpu_connection import IBMConnection
+
+circ1 = Circuit([Gate("H", 0), Gate("X", 1)])
+res_simulator_circ1 = {'01': 0.5, '11': 0.5}
+
+
+def assert_freq_dict_almost_equal(d1, d2, atol):
```

Comment:
I saw this function many places. I think it would be better to put it in `tangelo.helpers` (a testing module?) and import it from there.